### PR TITLE
Web repack script

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: engine/engine/wscript [hunk 1]
```diff
@@ -164,15 +164,10 @@ def build(bld):
...
-            filePath = os.path.join(srcPath, 'dmengine' + build + '.js.symbols')
-            if (os.path.exists(filePath)):
-                bld.install_files('${PREFIX}/bin/%s' % bld.env.PLATFORM, filePath)
-            filePath = os.path.join(srcPath, 'dmengine' + build + '.wasm')
-            if (os.path.exists(filePath)):
-                bld.install_files('${PREFIX}/bin/%s' % bld.env.PLATFORM, filePath)
+            for suffix in ('.js.mem', '.js.symbols', '.js.map', '.wasm', '.wasm.map'):
+                filePath = os.path.join(srcPath, 'dmengine' + build + suffix)
+                if (os.path.exists(filePath)):
+                    bld.install_files('${PREFIX}/bin/%s' % bld.env.PLATFORM, filePath)
```
Comment (JCash) [inline engine/engine/wscript:170]: Small cleanup of repetitive code

File: build_tools/waf_dynamo.py [hunk 2]
```diff
@@ -419,13 +419,21 @@ def default_flags(self):
...
 
+        flags = []
+        linkflags = []
+        if int(opt_level) < 2:
+            flags = ['-g4']
+            linkflags = ['-g4']
+
         for f in ['CCFLAGS', 'CXXFLAGS']:
-            self.env.append_value(f, ['-O%s' % opt_level, '-Wall', '-fPIC', '-fno-exceptions', '-fno-rtti',
+            self.env.append_value(f, ['-Wall', '-fPIC', '-fno-exceptions', '-fno-rtti',
```
Comment (JCash) [inline build_tools/waf_dynamo.py:429]: Removed duplicate flag

File: build_tools/waf_dynamo.py [hunk 3]
```diff
@@ -419,13 +419,21 @@ def default_flags(self):
         emflags = zip(['-s'] * len(emflags), emflags)
         emflags =[j for i in emflags for j in i]
 
+        flags = []
+        linkflags = []
+        if int(opt_level) < 2:
+            flags = ['-g4']
+            linkflags = ['-g4']
```
Comment (JCash) [inline build_tools/waf_dynamo.py:426]: Build with better symbols if we choose opt-level < 2 (2 is default)

File: scripts/web/web-repack.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+if [ -e "${DYNAMO_HOME}/bin/wasm-web/dmengine.wasm.map" ]; then
+    cp -v ${DYNAMO_HOME}/bin/wasm-web/dmengine.wasm.map ${TARGET}/dmengine.wasm.map
+fi
+
+# ----------------------------------------------------------------------------
+# Script teardown
+# ----------------------------------------------------------------------------
+trap - SIGHUP SIGINT SIGTERM EXIT
+echo "Wrote ${TARGET}"
+exit 0
```
Comment (JCash) [inline scripts/web/web-repack.sh:83]: Helps to update an existing html5 bundle

**Review by britzl — APPROVED** [review]

**Pull request merged**